src/client/internal.ts

```diff
@@ -81,3 +83,8 @@ export interface NotificationActionEventTransport {
     controlSource?: 'buttons';  // Additional sources will be added in future release
     controlIndex?: number;      // The index of the originating control, within notification[controlSource]
 }
+
+export interface ProviderStatus {
```

Comment:
Can you add doc comments to this? This will be in the client-facing API docs.

Reply:
✅ Doc'd and moved to the `provider` file.